app/views/content_items/service_manual_topic.html.erb

```diff
@@ -22,7 +22,11 @@
       <%= @content_item.description %>
     </p>
 
-    <div class="js-hidden" data-module="accordion-with-descriptions">
+    <% if @content_item.visually_collapsed? %>
```

Comment:
Realllllly don't like this approach here where we're outputting only one part of the div. I usually like to do something like:

```erb
<% container_class = @content_item.visually_collapsed? ? "js-hidden" : "" %>
<div class="<%=container_class%>">
</div>
```

```